RELEASE/scripts/sl_ascend/sl_mr2019.ash

```diff
@@ -159,6 +159,59 @@ boolean sl_sausageEatEmUp() {
 	return sl_sausageEatEmUp(0);
 }
 
+boolean sl_sausageGoblin()
+{
+	return sl_sausageGoblin($location[none], "");
+}
+
+boolean sl_sausageGoblin(location loc)
+{
+	return sl_sausageGoblin(loc, "");
+}
+
+boolean sl_sausageGoblin(location loc, string option)
+{
+	// Sausage Goblins have super low encounter priority so they will be overriden
+	// by all sorts stuff like superlikelies, wanderers and semi-rares.
+	// The good news is, being overridden just means adventure there again to get it
+
+	if(!possessEquipment($item[Kramco Sausage-o-Matic&trade;]))
+	{
+		return false;
+	}
+
+	// My (Malibu Stacey) and Ezandora's spading appears to guarantee the first 
+	// 7 sausage goblins using a formula of 3n+1 adventures since the previous.
+	// After that, you're on your own (hey it's better than nothing).
+	// Also that doesn't apply to the first goblin, it's always 100%.
+	int sausageFights = get_property("_sausageFights").to_int();
+	if (sausageFights >= 7)
```

Comment:
The condition in `offhand.dat` is _sausageFights>7, should they be consistent?

Reply:
You are correct & that is a bug. It would never get equipped in the current state other than for delay burning.